### DISCUSSION TITLE
How to relate dependencies to formulas?

### DISCUSSION THREAD

**Discussion by marmalodak** [Homebrew/discussions#967] [Everyday usage]
I'm trying to install [asciio](https://metacpan.org/pod/App::Asciio).  I've used brew to install the dependencies I recognize. Still need to find a few more.

    ❯ cpanm App::Asciio
    ...
    -> FAIL Finding Gtk2::Gdk::Keysyms (0) on mirror http://www.cpan.org failed.
    -> FAIL Couldn't find module or a distribution Gtk2::Gdk::Keysyms
    -> FAIL Installing the dependencies failed: Module 'Data::TreeDumper::Renderer::GTK' is not installed, Module 'Gtk2' is not installed, Module 'Gtk2::Gdk::Keysyms' is not installed, Module 'Glib' is not installed, Module 'Eval::Context' is not installed

What do I have do to install to satisfy the dependencies?  What is the process of relating these dependencies to formulas and casks?

**Comment by SMillerDev** [comment]
> What is the process of relating these dependencies to formulas and casks?

There is no process. You search for keywords and hope to find it. If upstream provides a list of Debian packages to install you can try repology.org to see what the homebrew equivalent is. But as far as I know there is no too to map an error from an install script directly to any package manager.
